esp/src/src-react/components/Sasha.tsx

```diff
@@ -62,36 +120,125 @@ export const Sasha: React.FunctionComponent<SashaProps> = () => {
   return (
     <div>
       <form onSubmit={handleSubmit}>
-        <Dropdown
-          placeholder={nlsHPCC.SelectAnOption}
-          selectedKey={selectedOption}
-          onChange={handleOptionChange}
-          options={[
-            { key: "", text: nlsHPCC.SelectAnOption },
-            { key: "getVersion", text: nlsHPCC.GetVersion },
-            { key: "getLastServerMessage", text: nlsHPCC.GetLastServerMessage },
-            { key: "restoreECLWorkUnit", text: nlsHPCC.RestoreECLWorkunit },
-            { key: "restoreDFUWorkUnit", text: nlsHPCC.RestoreDFUWorkunit },
-            { key: "archiveECLWorkUnit", text: nlsHPCC.ArchiveECLWorkunit },
-            { key: "archiveDFUWorkUnit", text: nlsHPCC.ArchiveDFUWorkunit },
-            { key: "backupECLWorkUnit", text: nlsHPCC.BackupECLWorkunit },
-            { key: "backupDFUWorkUnit", text: nlsHPCC.BackupDFUWorkunit }
-          ]}
-          styles={{ dropdown: { width: 400 } }}
-        />
-        {["restoreECLWorkUnit", "restoreDFUWorkUnit", "archiveECLWorkUnit", "archiveDFUWorkUnit", "backupECLWorkUnit", "backupDFUWorkUnit"].includes(selectedOption) && (
-          <div>
-            <TextField
-              label={nlsHPCC.WUID}
-              value={wuid}
-              onChange={(event: React.FormEvent<HTMLInputElement>, newValue?: string) => setWuid(newValue || "")}
-              styles={{ fieldGroup: { width: 400 } }}
-            />
-          </div>
-        )}
-        <PrimaryButton type="submit">{nlsHPCC.Submit}</PrimaryButton>
-        {defaultValue}
-        {result && <div>{nlsHPCC.Results}: {result}</div>}
+        <Stack tokens={{ childrenGap: 15, padding: 10 }}>
```

Comment:
This outermost &lt;Stack... is what's causing the submit button to be so much wider than all your other PRs for this page. It can be deleted without causing problems with any of the other fields.